tests/integration_tests/core/test_cli_response.py

```diff
@@ -148,3 +148,45 @@ def test_rasa_validate_quiet_no_errors(
     "you can use `rasa telemetry disable`." in output_text
     assert "To learn more, check out"
     "https://rasa.com/docs/rasa/telemetry/telemetry." in output_text
+
+
+def test_rasa_validate_null_active_loop_no_errors(
+    run: Callable[..., RunResult], request: FixtureRequest
+):
+    # Test captures the subprocess output for the command run
+    # and validates that the data in 'data/test/test_integration' throws no cli errors
+
+    test_data_dir = Path(request.config.rootdir, "data", "test", "test_integration")
```

Comment:
I don't see a stories file in this directory, is it missing?

Reply:
It's in the sub data directory. Added in line 161.